src/main/java/reposense/commits/model/CommitResult.java

```diff
@@ -29,6 +32,7 @@ public CommitResult(Author author, String hash, Date time, String messageTitle,
         this.tags = tags;
         this.insertions = insertions;
```

Comment:
Unrelated to this PR, but should we use assertions to check if insertions and deletions are non negative?

Reply:
the insertions and deletions are taken from git log, which they probably won't be negative. But for defensive programming, that would be good